lib/govuk_navigation_helpers.rb

```diff
@@ -1,5 +1,39 @@
 require "govuk_navigation_helpers/version"
 
 module GovukNavigationHelpers
-  # Your code goes here...
+  class NavigationHelper
+    def initialize(content_item)
+      @content_item = content_item
+    end
+
+    # Generate a breacrumb trail
+    #
+    # @return [Array<Hash>] Each item is a hash containing `:title` and `:url` for one link in the breadcrumb
+    def breadcrumbs
```

Comment:
I don't think it matters too much right now, the breadcrumb making is a relatively small function so I think it's simpler without an extra class. Can we leave it until we incorporate the related links part?